client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1782,13 +1782,12 @@
   "placeholder.search-by-notes": "Search by notes",
   "policyType.BUSINESS": "Business",
   "policyType.PERSONAL": "Personal",
-  "preference.allowTrackingOfReceivedStockByDonor": "Allow tracking of received stock by donor",
   "preference.displayPopulationBasedForecasting": "Display Population Based Forecasting",
   "preference.displayVaccinesInDoses": "Display vaccines in doses",
   "preference.manageVvmStatusForStock": "Manage VVM status for stock",
   "preference.showContactTracing": "Show Contact Tracing",
   "preference.sortByVvmStatusThenExpiry": "Sort available batches by VVM status then expiry",
-  "preference.allowTrackingOfReceivedStockByDonor": "Allow tracking of received stock by donor",
+  "preference.allowTrackingOfStockByDonor": "Allow tracking of stock by donor",
```

Comment:
Renamed to better reflect what the permission does... I know it's called allow tracking of received stock by donor, but we are allowing edits in stock and stocktake and the preference is now an OMS preference.... lemme know if you want it changed back (: